### PR TITLE
Full website completed

### PR DESCRIPTION
**Pull request by Wils-FOURR-speed**
### Project complete
This part of the project involved creating 3 pages of a website using **React Routers**
Besides that, it also had:

- Some personalized stylings
- Gitflow practices
- Descriptive README file
- React-redux linters to detect errors.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
-    </div>
+    <BrowserRouter>
+      <Navbar />
+
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<CalcDisplay />} />
+        <Route path="/quote" element={<ShowQuotes />} />
+      </Routes>
+    </BrowserRouter>
```
Comment (tresorsawasawa) [inline src/App.js:20]: - [ ] As required for the project, kindly ensure that general layout of your app matchs the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). Right now it is impossible to see `Navbar` on each page unless user scrols horizonataly.
### Your Design
Screenshot from 2023-08-16 03-27-07

### Expected wireframes
Screenshot from 2023-08-16 03-29-42

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Wils-FOURR-speed 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Wils-FOURR-speed) [reply src/App.js:20]: @tresorsawasawa Hi, please we were asked to personalize the website's styling.

Comment (peterrobert) [reply src/App.js:20]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Wils-FOURR-speed , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work personalising your ui ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**